### DISCUSSION TITLE
策略组配置代理集节点筛选不生效

### DISCUSSION THREAD

**Discussion by iamckun** [vernesong/OpenClash#3967] [Q&A]
![QQ_1721725932310](https://github.com/user-attachments/assets/a9668293-b4f3-4633-bd4e-c687076cafaf)
配置了筛选美国，其他没有满足条件的节点也进入了这个策略组

**Comment by yuntian123456** [comment]
解决了吗？

**Reply by iamckun** [reply]
> 解决了吗？

解决了但是忘记怎么弄的了现在不用这个了换了sing box

**Reply by yuntian123456** [reply]
好吧。。感谢回复。

**Reply by yuntian123456** [reply]
我明白了，原来是这样：
需要先设置有代理集才行
![image](https://github.com/user-attachments/assets/d8858619-acfc-43ef-ac88-d057bb839277)
![image](https://github.com/user-attachments/assets/db2133c1-5c36-48ca-8884-bb9cb36e0d5d)
然后在策略组配置代理集节点筛选就会生效。
太棒了，又发现一个全新玩法！
